WordPress/src/main/res/layout/activity_log_list_header_item.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We don't need the `FrameLayout` in this layout also.  We can just use the `WPTextView`.